src/web/wms/src/main/resources/GeoServerApplication.properties

```diff
@@ -46,7 +46,7 @@ KMLLayerConfigPanel.kmlFormatSettings           = KML Format Settings
 
 StyleEditPage.description = Edit the current style. The editor can provide syntax highlighting and automatic formatting. Click on the "validate" button to verify the style is a valid SLD document.
 StyleEditPage.name        = Name
-StyleEditPage.title       = Style Editor
+StyleEditPage.title       = Style Editor - {0}
```

Comment:
Ah, good catch, I did not notice. It seems there are two different template methods to override, getTitle and getPageTitle. I'll override both.